src/Dhall/Import.hs

```diff
@@ -321,6 +320,38 @@ instance Show MissingEnvironmentVariable where
         <>  "\n"
         <>  "↳ " <> Text.unpack name
 
+-- | Wrapper type for all Exceptions related to Imports
+data MissingImport = forall e. Exception e => MissingImport e
+  deriving (Typeable)
+
+instance Exception MissingImport
+
+instance Show MissingImport where
+  show (MissingImport e) = show e
+
+-- | List of Exceptions we encounter while resolving Import Alternatives
+newtype MissingImports = MissingImports [MissingImport]
```

Comment:
I think a simpler way to do this is to define `MissingImports` as:

```haskell
newtype MissingImports = MissingImports [SomeException]
```

... and then replace all occurrences of `MissingImport` with `Control.Exception.toException`

Reply:
Oh right, I'll try that 👍